pkg/spec/createconfig.go

```diff
@@ -106,19 +107,60 @@ type NetworkConfig struct {
 	PublishAll   bool     //publish-all
 }
 
+// SeccompPolicy determines which seccomp profile gets applied to the container.
+type SeccompPolicy int
+
+const (
+	// If set use SecurityConfig.SeccompProfilePath, otherwise use the default
+	// profile.  The SeccompProfilePath might be explicitly set by the user.
+	SeccompPolicyDefault SeccompPolicy = iota
+	// If set use SecurityConfig.SeccompProfileFromImage, otherwise follow
+	// SeccompPolicyDefault.
+	SeccompPolicyImage
+)
+
+// Map for easy lookups of supported policies.
+var supportedSeccompPolicies = map[string]SeccompPolicy{
+	"":        SeccompPolicyDefault,
+	"default": SeccompPolicyDefault,
+	"image":   SeccompPolicyImage,
+}
+
+// LookupSeccompPolicy looksup the corresponding SeccompPolicy for the specified
+// string. If none is found, an errors is returned including the list of
+// supported policies.
+// Note that an empty string resolved to SeccompPolicyDefault.
+func LookupSeccompPolicy(s string) (SeccompPolicy, error) {
+	policy, exists := supportedSeccompPolicies[s]
+	if exists {
+		return policy, nil
+	}
+
+	// Sort the keys first as maps are non-deterministic.
+	keys := []string{}
+	for k := range supportedSeccompPolicies {
+		keys = append(keys, k)
```

Comment:
Do we want to include the empty string here? It's a valid key, but I don't know if we want it in the help dialogue

Reply:
Good catch! Fixed now.